docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,114 @@
+
+Users are still responsible for their keys, but through account abstraction, they can take
+precautions to ensure they do not end up losing their accounts.
+
+Account abstraction introduces a layer of on-chain logic that controls an account, typically in the
+form of a smart contract. Without a smart contract, abstracting accounts would require changes in
+the core architecture of the protocol.
+
+The highly generic codebase of Polkadot's architecture makes accounts natively flexible and abstract
+without the direct need for smart contracts.
+
+## Origin Abstraction in Polkadot
+
+Adopting a generic design is crucial in scaling Web3 technologies. Abstraction and generalization of
+protocols are essential to improving user experience and security in blockchain adoption.
+
+When users interact with a blockchain they call _dispatchable_ functions to do something. Because
+those functions are called from the outside of the blockchain interface, in Polkadot's terms any
+action that involves a dispatchable function is an [extrinsic](./learn-extrinsics.md). Extrinsics
+are calls coming from the _outside_ of the blockchain interface that (if successfully executed)
+invoke some changes in the _inside_ of a blockchain, the blockchain's state. An extrinsic is always
+directed to a specific function within a particular pallet (or module).
+
+For example, the `balances.transferKeepAlive` extrinsic is directed to the `transferKeepAlive`
+function within the `balances` pallet. If successful, the execution of that function will transfer
+funds between two accounts, changing the balances of those accounts and thus the chain state (as
+accounts hold some state within the blockchain).
+
+In [Substrate](../general/glossary.md#substrate)'s FRAME, functions are not necessarily called by
+accounts. Functions can be called by any origin, where origins are caller-personas associated with
+privilege levels. For example, the Polkadot [OpenGov](./learn-polkadot-opengov.md) has different
+origins with different privileges, such as allocating treasury funds, cancelling a referendum, etc.
+Neither of those origins is subservient to the concept of an account or assume anything about state
+or associated data. Custom origins can be created while designing your chain using the
+[Polkadot SDK](https://github.com/paritytech/polkadot-sdk).
+
+**Accounts happen to be just one variant (or corner case) of Substrate's FRAME possible origins, the
+`frame_system::RawOrigin::Signed`.** In Substrate, the concept of account is completely
+deprioritized. Substrate itself remains indifferent to an account's balance and nonce. While FRAME
+can support their presence, it fundamentally does not need to rely on them.
+
+On a lower level, the [Cross-Consensus Messaging (XCM)](./learn-xcm-index) format provides a much
+powerful origin abstraction that allows calling personas that are so abstract to not necessarily
+have direct representation on the local chain within its FRAME system origin.
```

Comment:
```suggestion
On a lower level, the [Cross-Consensus Messaging (XCM)](./learn-xcm-index) format can take advantage of origin abstraction for cross-consensus communications by specifying the context for a particular message.  Origins in this case imply the authority under which a message is being sent (and thereby, executed).
```

Reply:
added together with the previous paragraph about XCM